C++/0. Setup.md

```diff
@@ -22,18 +27,19 @@ int main() {
 
 ## Comment
 
-In the code before I wrote a comment. A comment in C++ is written with a `// `.
+In the code before I wrote a comment. A comment in C++ is written with a `// ...` or `/* ... */`, where `...` takes place of the comment's contents.
+Comments are regularly used to document a piece of code's functionality, e.x.: why a decision was made, or why something is implemented in a particular way.
 
 ### Code
 
 ```cpp
 int main() {
-  // Tis is a comment
+  // This is a comment
```

Comment:
Was the "Tis" intended? If so, I can revert.

Reply:
well I have my own way of saying things in a funny way but keep it as this as that makes more sense.